pax-web-tomcat/src/main/java/org/ops4j/pax/web/service/tomcat/internal/EmbeddedTomcat.java

```diff
@@ -215,7 +215,8 @@ void configure(Configuration configuration) {
 			tomcatResource = getClass().getResource("/tomcat-server.xml");
 		}
 
-		File configurationFile = new File(configuration.getConfigurationDir(),
+		configurationDir = configuration.getConfigurationDir();
+		File configurationFile = new File(configurationDir,
 				SERVER_CONFIG_FILE_NAME);
```

Comment:
In the beginning I also had a file handler in the jetty part, but how do you use it in case it's a fragment?
I think a URI could be better for this